src/Apps/Order/Components/SavedAddressItem.tsx

```diff
@@ -19,7 +19,7 @@ interface SavedAddressItemProps {
 
 export const SavedAddressItem: React.FC<SavedAddressItemProps> = (
   props
-): React.ReactElement<RadioProps> => {
+): React.ReactElement<RadioProps<string>> => {
```

Comment:
@erikdstock hi! Could you, please, confirm that I am using the right type here?
 The radio button component on Palette was changed and we need to specify the value type now